recipes/gdal2/build.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+if [ `uname` == Darwin ]; then
+    PGFLAG=""
+    export LDFLAGS="-headerpad_max_install_names"
+#    export CC=clang
+#    export CXX=clang++
+else
+    PGFLAG="--with-pg=$PREFIX/bin/pg_config"
+fi
+
+CPPFLAGS="-I$PREFIX/include" LDFLAGS="-L$PREFIX/lib" \
+./configure --prefix=$PREFIX \
+--with-geos=$PREFIX/bin/geos-config \
```

Comment:
> tests go in run_test.py, right?

Yes.  Just something simple to ensure that the desired feature is there.

> Ähm, I'm not sure, this is verbatim from conda-recipes repo.

Yes. I see that now.  Sorry.